examples/py_file/src/main.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 MesaTEE Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use lazy_static::lazy_static;
+use mesatee_sdk::{Mesatee, MesateeEnclaveInfo};
+use std::net::SocketAddr;
+use std::{env, fs};
+
+static FUNCTION_NAME: &str = "mesapy_from_buffer";
+static USER_ID: &str = "uid";
+static USER_TOKEN: &str = "token";
+
+lazy_static! {
```

Comment:
should we use mesatee_config here? i'm not sure

Reply:
No, `mesatee_config` is for `core`, `services`, etc. This is an example, hard-coded addresses are intuitive for others reading the code. Other examples are also using the same definitions here.